apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -398,15 +398,18 @@ const Ticket = new GQLListSchema('Ticket', {
     },
     plugins: [uuided(), versioned(), tracked(), softDeleted(), dvAndSender(), historical()],
     hooks: {
-        resolveInput: async ({ operation, listKey, context, resolvedData, existingItem }) => {
-            await triggersManager.executeTrigger({ operation, data: { resolvedData, existingItem }, listKey, context }, context)
+        resolveInput: async ({ operation, context, resolvedData, existingItem }) => {
             // NOTE(pahaz): can be undefined if you use it on worker or inside the scripts
             const user = get(context, ['req', 'user'])
             const userType = get(user, 'type')
             const newItem = { ...existingItem, ...resolvedData }
             const resolvedStatusId = get(newItem, 'status', null)
             const resolvedClient = get(newItem, 'client', null)
 
+            if (operation === 'create' && !resolvedStatusId) {
```

Comment:
Is resolvedStatusId can take other values than `null`? If not I think it's worth checking more explicitly 
```suggestion
            if (operation === 'create' && !isNull(resolvedStatusId)) {
```
or
```suggestion
            if (operation === 'create' && resolvedStatusId === null) {
```